libbeat/processors/rate_limit/clock/clock.go

```diff
@@ -0,0 +1,27 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Should we move this package under `libbeat/common`?

Reply:
I ended up using https://github.com/jonboulle/clockwork. Implemented in f597e5f96.